mirage/config.js

```diff
@@ -214,7 +214,38 @@ export default function configure() {
     return {};
   });
 
-  this.get('/packages/:id/resources', nestedResourceRouteFor('package', 'resources'));
+  let packageTitlesFilter = (resource, req) => {
```

Comment:
I would expect this to look more like the title search route below. For example, `mod-kb` would accept `filter[subject]=somesubject` instead of `searchfield=subject&q=somesubject`.

For the other related PR, I copied the existing package search route filter and used it in both package search and provider package search. And that's exactly what the backend does as well (uses the same filters & params for top-level and related searches)